src/main/java/HTML1/Task2.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+    <style>
+
+        body {
+            background-color: PeachPuff;
+        }
+
+        .box {
+            border: 3px solid red;
+            background-color: black;
+            padding: 40px;
+            margin: 40px;
+            width: 500px;
+            height: 130px;
+        }
+
+        .box1 {
+            border: 3px solid red;
+            background-color: black;
+            padding: 40px;
+            margin: 40px;
+            width: 500px;
+            height: 170px;
```

Comment:
Can remove this property (height) and `.box` and `.box1` will become the same, so it allow us to remove one of this classes, which is actually useless